osquery/config/parsers/prometheus_targets.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <iostream>
+
+#include <osquery/config.h>
+#include <osquery/logger.h>
+
+#include <osquery/config/parsers/prometheus_targets.h>
```

Comment:
nit: Use `"` for relative path to includes:
`#include "osquery/config/parsers/prometheus_targets.h"`